code/datums/traits/good.dm

```diff
@@ -39,6 +54,21 @@
 
 
 
+/datum/quirk/heat_res
+	name = "Heat-Defying"
+	desc = "You're used to hot environments. You take less damage from intense heat, but slightly more damage from cold"
+	value = 1
+	gain_text = "<span class='notice'>You feel at ease in the heat.</span>"
+	lose_text = "<span class='notice'>You no longer feel at ease in the heat.</span>"
+
+/datum/quirk/heat_res/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/datum/species/species = H.dna.species
+	species.heatmod = species.heatmod - 0.2
+	species.coldmod = species.coldmod - 0.1
```

Comment:
am i reading this wrong or are one of these being applied incorrectly

Reply:
you're not wrong, last minute oversight